core/unsafe/console.js

```diff
@@ -33,16 +37,46 @@ colours.setTheme({
     title: ['green', 'bold']
 });
 
+var getTimestampString = function() {
+	var dt = new Date(),
+		diff = (dt.getTime() / 1000) - startupTime,
+		time = '[' + ('          ' + diff.toFixed(2)).slice(-10) + '] ';
```

Comment:
As for just printing a date/time instead of an uptime there are a few reasons for doing that:
- uptime is more useful to us, we can see precisely how long it took to get to an error, timings between problems, etc
- the original time can still be calculated, the start time is printed at the beginning of the log
- logs get really really convoluted and messy with a full DateTime string at the beginning, uptime is short and easily readable
- it is consistent with other software e.g. dmesg